cmd/query/app/server_test.go

```diff
@@ -49,6 +50,8 @@ import (
 
 var testCertKeyLocation = "../../../pkg/config/tlscfg/testdata"
 
+var tracer = jtracer.OT(opentracing.NoopTracer{})
```

Comment:
this becomes a global variable in the package used in other test files. So at minimum it should have a more descriptive name, e.g. `noopTracer`.

Perhaps a better solution is to add a function NoOp() to the `jtracer` package, and not introduce global var at all

Reply:
I create `NoOp()` to the jtracer package.